lab-elizabeth/app/scss/main.scss

```diff
@@ -0,0 +1,68 @@
+@import '_vendor';
+@import '_theme';
```

Comment:
Duncan mentioned this today in class, but you don't need the underscores when importing scss partials.